src/SFML/Graphics/RenderStates.cpp

```diff
@@ -35,24 +35,35 @@ namespace sf
 ////////////////////////////////////////////////////////////
 // We cannot use the default constructor here, because it accesses BlendAlpha, which is also global (and dynamically
 // initialized). Initialization order of global objects in different translation units is not defined.
-const RenderStates RenderStates::Default(BlendMode(
-    BlendMode::SrcAlpha,
-    BlendMode::OneMinusSrcAlpha,
-    BlendMode::Add,
-    BlendMode::One,
-    BlendMode::OneMinusSrcAlpha,
-    BlendMode::Add));
+// For similar reasons we need to provide the default stencil mode, and thus all other settings along with it
+const RenderStates RenderStates::Default(
+    BlendMode(BlendMode::SrcAlpha,
+              BlendMode::OneMinusSrcAlpha,
+              BlendMode::Add,
+              BlendMode::One,
+              BlendMode::OneMinusSrcAlpha,
+              BlendMode::Add),
+    StencilMode(StencilMode::Always, StencilMode::Keep, 0, ~0u, false),
```

Comment:
The `StencilMode` default constructor already uses these same values. Why don't we just default-initialize this member variable instead?